tests/unit/test_scm.py

```diff
@@ -24,6 +24,10 @@ class TestRepoID:
             # no-op
             ("ssh://user@git.host.com/some/path", "ssh://user@git.host.com/some/path"),
             ("https://git.host.com/some/path", "https://git.host.com/some/path"),
+            (
```

Comment:
Just a tiny nitpick: the new test scenario is under a comment saying `#no-op`, but in this case, we're actually changing the provided url. So I'd add a new comment to separate it into a new category.